packages/react/src/MeasureReportDisplay/MeasureReportGroup/MeasureReportGroup.tsx

```diff
@@ -0,0 +1,138 @@
+import { Box, Flex, Group, Paper, RingProgress, Text, Title } from '@mantine/core';
+import { formatCodeableConcept } from '@medplum/core';
+import { Measure, MeasureReportGroup } from '@medplum/fhirtypes';
+import React from 'react';
+import { QuantityDisplay } from '../../QuantityDisplay/QuantityDisplay';
+
+interface MeasureGroupProps {
```

Comment:
Can we rename the props interface to match the component name? `MeasureReportGroupDisplayProps`.  I know it's a mouthful, but I think it's "technically correct".